Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2_test.py

```diff
@@ -1911,34 +1920,6 @@ def test_get_ids_with_retry_success(self, mock_parse, mock_diff, mock_demisto):
             assert result_ids == ["id1", "id2"]
 
 
```

Comment:
It seems the test case for handling an empty/failed response (`test_test_response_empty_response`) was removed but not replaced. To ensure the error handling path of `check_response` is covered, could you please add a test for this failure scenario?